### PR TITLE
Peer Reviews

### PR DESCRIPTION
**Pull request by yattias**
Solves #1167 

### Creating
image

### Top
image

### Feed
image

### Discussion
image

### Profile
image

### CODE REVIEW

File: components/Form/ScoreInput.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      setSelectedValue(value);
+    }
+  }, [value])
+
+  const handleSelect = (value) => {
+    setSelectedValue(value);
+    onSelect && onSelect(value);
+  }
+
+  const buildBarInput = () => {
```
Comment (calvinhlee23) [inline components/Form/ScoreInput.tsx:40]: was there a specific reason why this was a functional? 
If not, let's just relocate this to be inside of useMemo. Or maybe making this its own component outside of  `ScoreInput` is better (I understand that keep it inside is convenient for variable access)

File: components/Form/ScoreInput.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
+import { css, StyleSheet } from "aphrodite";
```
Comment (calvinhlee23) [inline components/Form/ScoreInput.tsx:1]: hmm the conditionals-formatting looks off in this file. What kind of linter are you using?

File: components/Form/Toggle.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+function Toggle({
+  options,
+  onSelect,
+  selected,
+}: Props): ReactElement {
+
+  const handleSelect = (opt:ToggleOption) => {
+    onSelect(opt);
+  }
```
Comment (calvinhlee23) [inline components/Form/Toggle.tsx:25]: looks like this func isn't doing anything. 
Let's remove it and if we see the need to have its own handleSelect function, we can add back in(?)

File: components/Form/Toggle.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  selected: any;
+}
+
+type Props = {
+  options: Array<ToggleOption>;
+  onSelect: Function;
+  selected: any;
+};
+
+function Toggle({
```
Comment (calvinhlee23) [inline components/Form/Toggle.tsx:17]:
```suggestion
export default function Toggle({
```

File: components/Form/Toggle.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    }
+  },
+  "optionSelected": {
+    background: colors.NEW_BLUE(),
+    color: "white",
+    transition: "0.1s",
+  }
+});
+
+export default Toggle;
```
Comment (calvinhlee23) [inline components/Form/Toggle.tsx:68]:
```suggestion
```

File: components/Paper/Tabs/DiscussionTab.js [hunk 6]
```diff
@@ -7,19 +7,23 @@ import Ripples from "react-ripples";
 import ReactPlaceholder from "react-placeholder";
```
Comment (calvinhlee23) [inline components/Paper/Tabs/DiscussionTab.js:7]: Thanks for starting to refactor this. 
I've already created ticket for this for both FE & BE. 
But let's really consider migrating away from the old commenting modules.
Anytime, this component / BE needs to be updated for these, it's too much effort & I don't think it's going to be scalable the way we have them.

File: components/PeerReviews/PeerReviewScoreSummary.tsx [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+  return (
+    <div className={css(styles.reviewContainer)}>
+      <span className={css(styles.reviewScoreContainer)}>
+        <span className={css(styles.reviewScore)}>{summary?.avg}</span>/10
+      </span>
+      <div className={css(styles.scoreContainer)}>
+        <span className={css(styles.dot)}>&bull;</span>
+        <ScoreInput
+          value={summary?.avg}
+          readOnly={true}
```
Comment (calvinhlee23) [inline components/PeerReviews/PeerReviewScoreSummary.tsx:27]: This evaluates to `true`
```suggestion
          readOnly
```

**Review by calvinhlee23 — CHANGES_REQUESTED** [review]
Thoughts and comments.

File: components/Form/ScoreInput.tsx (hunk 2 above)
Comment (yattias) [reply components/Form/ScoreInput.tsx:1]: eslint. not sure why it skipped. Could be that it doesn't do space formatting?
I actually don't see spaces/tabs defined in eslint

File: components/Form/ScoreInput.tsx (hunk 1 above)
Comment (yattias) [reply components/Form/ScoreInput.tsx:40]: sounds reasonable. I think I had a reason earlier on.

File: components/Form/Toggle.tsx (hunk 3 above)
Comment (yattias) [reply components/Form/Toggle.tsx:25]: yeah, I had it do something else at some point. I'll clean it up

**Pull request merged**